dashboard/app/controllers/activities_controller.rb

```diff
@@ -218,15 +219,15 @@ def trophy_check(user)
   end
 
   def log_milestone(level_source, params)
-    log_string = "Milestone Report:"
-    if (current_user || session.id)
-      log_string += "\t#{(current_user ? current_user.id.to_s : ("s:" + session.id))}"
+    log_string = 'Milestone Report:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.